Jenkinsfile

```diff
@@ -0,0 +1,143 @@
+#!groovy
+/*
+ * This Jenkinsfile is intended to run on https://ci.jenkins-ci.org and may fail anywhere else.
+ * It makes assumptions about plugins being installed, labels mapping to nodes that can build what is needed, etc.
+ */
+
+// TEST FLAG - to make it easier to turn on/off unit tests for speeding up access to later stuff.
+def runTests = true
+
+// Only keep the 10 most recent builds.
+properties([[$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator',
+                                                          numToKeepStr: '10']]])
+
+// TODO: Once https://github.com/jenkinsci/packaging/pull/34 is merged, switch to master branch.
```

Comment:
If you reference this pull request URL in the commit message or pull request message, GitHub will link them up.